### ISSUE TITLE
test fails on armhf/i386: -{"hex3":3735928559}, +{"hex3":-559038737}

### ISSUE THREAD

**Issue by bdrung** [komkom/toml#5]
The test suite fails on 32-bit architectures (like armhf and i386):

```
    reader_test.go:184: 
        	Error Trace:	reader_test.go:184
        	Error:      	Not equal: 
        	            	expected: "{\"hex3\":3735928559}"
        	            	actual  : "{\"hex3\":-559038737}"
        	            	
        	            	Diff:
        	            	--- Expected
        	            	+++ Actual
        	            	@@ -1 +1 @@
        	            	-{"hex3":3735928559}
        	            	+{"hex3":-559038737}
        	Test:       	TestReader
```

Bug-Debian: https://bugs.debian.org/1004048

**Comment by komkom** [comment]
The issue should be resolved with this PR https://github.com/komkom/toml/pull/6.
Could you double check?

**Comment by bdrung** [comment]
Yes, PR #6 fixes the test failure. While testing the fix, I stumbled over #7.
